pkg/beholder/client.go

```diff
@@ -256,17 +257,16 @@ func newTracerProvider(config Config, resource *sdkresource.Resource, creds cred
 	if err != nil {
 		return nil, err
 	}
-	tp := sdktrace.NewTracerProvider(
-		sdktrace.WithBatcher(exporter,
-			trace.WithBatchTimeout(config.TraceBatchTimeout)), // Default is 5s
+	opts := slices.Clone(config.TracerProviderOpts)
+	opts = append(opts, sdktrace.WithBatcher(exporter,
```

Comment:
@jmank88 Probably not safe to just append it like this. Some of these options are mutually exclusive. 
Probably better to not mix them up and use the default options from Beholder or use overrides from the config